### ISSUE TITLE
Sprint 12 Goal

### ISSUE THREAD

**Issue by hmitsch** [mozilla-iam/iam-project-backlog#175]
## Prioritized Themes
1. RP Cutovers
1. IAM Q2 OKR _PROD transition_ (Mozillians/CIS/Discourse) 
1. Mozillians.org Infrastructure and Application _debt repayment_

### Details
1. RP Cutover 
    - Migration from Okta to Auth0 SSO, see [cut over verification link](https://github.com/mozilla-iam/sso-dashboard-configuration/blob/master/apps.yml)
    - [Migration Plan](https://docs.google.com/spreadsheets/d/1VLwh2A9qbaRqByj7BYbnziWQLL_DdjemAiwT4_0OebE/edit)
    - [ ] Group 1 (all managed by IT) #131 
    - [ ] Group 3 (Cutover coordinated independently) #133 
    - [ ] Decom RPs that won't migrate #61
1. Change Integration Service ([CIS](https://github.com/mozilla-iam/cis/issues)): _Q2 OKR PROD transition_
    - [ ] RP Handle new group information #116 (In progress)
    - [ ] Filter users interacted with https://github.com/mozilla-iam/cis/issues/5
    - [ ] Merge demo functionality of CIS to mozillians.org
    - [ ] Moz/CIS integration #107 (in progress)
    - [ ] CIS clean-up activities: #118, #124, Back-up/recovery of dynamo
    - [ ] Event format specification document for publisher (@andrewkrug @gdestuynder)
    - [ ] Cross account role permission for CIS prod between mozillians.org and infosec prod
    - [ ] Create production KMS key for event signing (@gene1wood @andrewkrug)
1. Mozillians.org  _Infrastructure Cleanup_
   - [x] Infra Maintenance https://github.com/mozilla/parsys/issues/202
   - [ ] Fix mozillians.org S3 permissions https://github.com/mozilla/parsys/issues/134 https://github.com/mozilla/parsys/issues/191
   - [ ] Remove memcached from mozillians group https://github.com/mozilla/parsys/issues/190
   - [ ] Add cdn to mozillians media https://github.com/mozilla/parsys/issues/189
   - [x] Move staging apps to prod cluster https://github.com/mozilla/parsys/issues/133 
   - [ ] Work on open issues from [Mozillians.org Penetration Test](https://docs.google.com/document/d/19xD3UlDgb_vQJYVX0g3pvqfg1a3O6HjIsnvnC2vg-kI/edit)
      - [ ] [MOZILLIANS.ORG] Connect auditd to mozdef https://github.com/mozilla/parsys/issues/93
https://github.com/mozilla/parsys/issues/158
1. Mozillians.org _Application Cleanup_
   - [ ] Mozillians.org repo/Github org (debt) cleanup
   - [ ] Spamfiltering: get it to DONE-DONE, i.e. enable automatic account removal
   - [ ] Binary search: get it to DONE-DONE
   - [ ] [Bug 1373894](https://bugzilla.mozilla.org/show_bug.cgi?id=1373894): Potential XSS in Mozillians.org
    - [ ] [Bug 1376416 mozillians.allizom.org and mozillians-dev.allizom.org  are unreachable - 403 forbidden](https://bugzilla.mozilla.org/show_bug.cgi?id=1376416)
    - [ ] Implement a way to trigger reindexes https://github.com/mozilla/parsys/issues/201
    - [ ] Celery upgrade to version 4 for Mozillians https://github.com/mozilla/parsys/issues/210
1. QA Engineering
   - [x] Set the login tests to use counter-based OTPs #176 
   - [x] Implement login tests for sso.mozilla.com, mozillians.org, standu.ps, discourse, mozillastage.service-now.com: #179 , #181 , #177 , #180 , #178.  
   - [x] Set the login tests to run in parallel in Jenkins #182 
   - [ ] (bonus) Automate client testing in auth0-deploy CI, include SAML clients https://github.com/mozilla/iam-project-backlog/issues/162
1. [SSO Dashboard] Launch for Communities
   - [ ] Communicate the publicly available Dashboard widely: Discourse, various Telegram Channels, mailto all Mozillians (via Salesforce Marketing Cloud), Mozilla Reps

## For mental sanity: Other InfoSec Work
1. "Second Opinion" #79 
    - [ ] Productionalize
1. SSH Federated Access (multi-sprint) #169 
    - [ ] Finalize project documentation (https://mana.mozilla.org/wiki/display/SECURITY/Auth0+and+SSH)
    - [ ] POC Client-side 
    - [ ] POC Server-side
    - [ ] POC IT Deploy with original groups
1. Accommodate Auth0 OIDC Conformant mode requirements
    - [ ] Migrate all RPs to full OIDC conformant mode #171

## For mental sanity: Other ParSys Work
1. [x] [Bug 1382208](https://bugzilla.mozilla.org/show_bug.cgi?id=1382208) - "Nominate Rep of the Month" task not expired
1. [x] [Bug 1381890](https://bugzilla.mozilla.org/show_bug.cgi?id=1381890) - [Basket] Add double opt-in parameter to email subscription API calls & clean up existing Mozillians/NDA subscribers
1. [CoSS](https://github.com/mozilla/coss/) Sprint 12 https://github.com/mozilla/coss/issues/216
1. [Discourse] https://github.com/mozilla/discourse/milestone/1

## Unplanned Scope
**Note:** please add any unplanned scope that you worked on here.
* [Bug 138342](https://bugzilla.mozilla.org/show_bug.cgi?id=1383642) - OVH servers are down
## Out of Scope / Later Items
Themes which are not **yet** slated for delivery
* [Fix bug 1201027] Simplify Profile status UI https://github.com/mozilla/mozillians/pull/1436
* Groups grid UI enhancements https://github.com/mozilla/mozillians/pull/1426/
* (Mentoring) [Bug 847911](https://bugzilla.mozilla.org/show_bug.cgi?id=847911) Create a view with all vouched mozillians
* (Mentoring) [Bug 1333373](https://bugzilla.mozilla.org/show_bug.cgi?id=1333373) Redesign footer based on new brand identity
* Automated ServiceNow ticket creation for SSO Dashboard deployments
* Implement API Key Support on behalf of RPs
* SSO Dashboard: Implement low- and med-risk alert UI https://github.com/mozilla-iam/sso-dashboard/issues/130
* Slack for Community access (next iteration of the current manual _NDA Pilot_)
* WorkDay/CIS integration
* [SSO Dashboard] (bugfix) Run access-proxy in front of SSO dashboard (https://github.com/mozilla-iam/sso-dashboard/issues/107)
* Move AirMo to 'hosted Auth0 Lock'
* SSO Dashboard improvements
   - UX: user favorites https://github.com/mozilla-iam/sso-dashboard/issues/92 / frequently used #98
   - Fix SSO with ParSys properties
* [AUTH0 LOCK] _Prototype new technology stack_
   - Sign Up UI design #67 
   - Decide on technology stack for Front-end and AWS Lambda (backend)
   - Create backlog items

## Logistics
* Consulted: https://github.com/mozilla-iam/iam-project-backlog/issues/173
* Out of Office: @comzeradd out for most of the sprint
* CoSS share:  @m-branson and @akatsoulas focus mostly on CoSS
